addons/project/tests/test_multicompany.py

```diff
@@ -272,7 +272,7 @@ def test_create_subtask(self):
                     node.attrib.pop('attrs')
                 view.arch = etree.tostring(tree)
                 with self.debug_mode():
-                    with Form(self.task_2) as task_form:
+                    with Form(self.task_2.with_context(uid=self.env.uid)) as task_form:
```

Comment:
Adding `uid` in the context does not look right.  Is the web client doing that _for evaluating a domain_?  If that's the case, it should be in the implementation of the form view itself, not in the code of the test.